vm-virtio/src/console.rs

```diff
@@ -31,8 +31,26 @@ const INPUT_QUEUE_EVENT: DeviceEventT = 0;
 const OUTPUT_QUEUE_EVENT: DeviceEventT = 1;
 // Some input from the VMM is ready to be injected into the VM.
 const INPUT_EVENT: DeviceEventT = 2;
+// Console configuration change event is triggered.
+const CONFIG_EVENT: DeviceEventT = 3;
 // The device has been dropped.
-const KILL_EVENT: DeviceEventT = 3;
+const KILL_EVENT: DeviceEventT = 4;
```

Comment:
Would have been better to make CONFIG_EVENT  =4

Reply:
Done.